contracts/VetoGuard.sol

```diff
@@ -112,6 +112,14 @@ contract VetoGuard is
         emit TransactionQueued(msg.sender, transactionHash, signatures);
     }
 
+    /// @notice Updates the execution delay blocks, only callable by the owner
+    /// @param _executionDelayBlocks The number of blocks between when a transaction is queued and can be executed
+    function updateExecutionDelayBlocks(
+        uint256 _executionDelayBlocks
+    ) external onlyOwner {
+        executionDelayBlocks = _executionDelayBlocks;
```

Comment:
Is there any check to make sure that the `executionDelayBlocks` cannot be set to less than the length of a potential freeze / veto vote?

Reply:
There isn't a check for this currently. However, the veto and freeze voting doesn't have a minimum time that they would take to succeed. And the amount of time the parent DAO has to pass a veto proposal is only constrained by that `executionDelayBlocks` variable in the `VetoGuard`